### PR TITLE
feat: four eyes principle

### PR DESCRIPTION
**Pull request by cakarci**

### CODE REVIEW

File: __tests__/pull-request-workflow.test.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import {PullRequestWorkflow} from '../src/pull-request-workflow'
-import {getFileContent, getRandomListItems} from '../src/utils'
+import {getFileContent, getRandomItemFromArray} from '../src/utils'
 
 jest.mock('../src/utils', () => ({
   getFileContent: jest.fn().mockResolvedValue(''),
-  getRandomListItems: jest.fn()
+  getRandomItemFromArray: jest.fn()
```
Comment (pcakarci) [inline __tests__/pull-request-workflow.test.ts:6]: good change!

**Review by pcakarci — APPROVED** [review]

File: __tests__/pull-request-workflow.test.ts (hunk 1 above)
Comment (pcakarci) [reply __tests__/pull-request-workflow.test.ts:6]: good change!

**Review by pcakarci — APPROVED** [review]

File: __tests__/pull-request-workflow.test.ts (hunk 1 above)
Comment (pcakarci) [reply __tests__/pull-request-workflow.test.ts:6]: good change!

**Review by pcakarci — APPROVED** [review]

File: __tests__/pull-request-workflow.test.ts [hunk 2]
```diff
@@ -22,6 +22,6 @@ describe('Pull request workflow', () => {
   it('should call getFileContent and getRandomListItems', async () => {
     await PullRequestWorkflow()
     expect(getFileContent).toHaveBeenCalled()
-    expect(getRandomListItems).toHaveBeenCalled()
+    expect(getRandomItemFromArray).toHaveBeenCalled()
```
Comment (pcakarci) [inline __tests__/pull-request-workflow.test.ts:25]: better to return one item

**Review by pcakarci — APPROVED** [review]

**Review by pcakarci — APPROVED** [review]

**Review by pcakarci — APPROVED** [review]
test

**Review by pcakarci — APPROVED** [review]

**Review by scakarci — APPROVED** [review]

**Review by scakarci — APPROVED** [review]

**Review by scakarci — APPROVED** [review]

**Review by pcakarci — APPROVED** [review]

**Review by scakarci — APPROVED** [review]

**Review by pcakarci — APPROVED** [review]

**Review by pcakarci — APPROVED** [review]

**Review by scakarci — APPROVED** [review]

**Review by pcakarci — COMMENTED** [review]
aaaa

**Review by pcakarci — APPROVED** [review]

**Review by scakarci — APPROVED** [review]

**Pull request closed without merging**